atlp-rwanda/ecommerce-app-champions-bn
# #44 Email and in-app notifications

# What this PR do
- Create email and in-app notifications
## Description of the  task to be completed

- To send email notifications  to the product lifecycle changes
- To send app notifications to the product lifecycle changes 

## How should this be manually tested?

- git clone from repo
- switch into branch
- run npm i
- ## How should this be manually tested
- clone the repo
- use npm install to install all packages
- Add all required environment variables following the .env.example provided
- use npx sequelize db:migrate to migrate migrations
- Run npm run db:seeds to populate seeders in database
- use npm run dev to start the server
- Navigate to  the http://localhost:PORT/docs on your browser to access swagger APIs docs  by replacing 'PORT' with the port that your the app is listening to. 
- run npm run dev
- try to add product
- check your email  and app for notification:
- Trello ID: #44
